core/src/main/java/org/nervos/ckb/transaction/LockUtils.java

```diff
@@ -17,8 +16,7 @@ public static Script generateLockScriptWithPrivateKey(String privateKey, byte[]
   }
 
   public static Script generateLockScriptWithAddress(String address) {
```

Comment:
I think this fn has been replaced by `Address.decode(...).getScript()`, please remove it.